introducao_curso/main.tex

```diff
@@ -1,18 +1,21 @@
+Na UFSCar utilizamos o sistema de créditos - você pode entender créditos como horas semanais de um semestre - em que, no curso Ciência da Computação você precisará de 217 créditos para se formar sendo eles: 109 de disciplinas obrigatórias, 56 de optativas, 24 do Trabalho de Graduação ou Estágio, 22 créditos de atividades de extensão e 6 créditos de atividades complementares. Como aluno, é necessário que você complete pelo menos 8 créditos em um ciclo de um ano (contabilizando o semestre atual com o semestre anterior). Caso esse número de créditos não seja atingido, o estudante é considerado ``jubilado'' e sua matrícula é cancelada, perdendo a vaga. Além deste requisito, o calouro tem que passar em pelo menos 4 créditos no primeiro semestre para não perder a vaga. Isso pode ser reversível com a utilização do recurso (e é claro, uma justificativa convincente). Por isso muito cuidado!
 
-O IRA é o Índice de Rendimento Acadêmico e é adotado para classificação na concorrência de prioridade de inscrição de disciplinas, transferências internas, bolsas para iniciação científica e até para programas externos como Ciência Sem Fronteiras. 
+O IRA é o Índice de Rendimento Acadêmico e é adotado para classificação na concorrência de prioridade de inscrição de disciplinas, transferências internas e bolsas para iniciação científica.
 
-Abaixo temos um ilustrativo de como serão os três primeiros semestres dos ingressantes. As setas cheias representam quais matérias precisam ter aprovações para se cursar as outras, já as setas pontilhadas são aquelas que são recomendadas.
+Abaixo temos um ilustrativo de como serão os quatro primeiros semestres dos ingressantes. As setas representam quais matérias precisam ter aprovações para se cursar as outras.
 
 O que acontece se você por acaso não passar em uma matéria? Ele fica com a
 famosa dependência ou DP, tendo de refazê-la em um semestre futuro. Note a
-importância de Algoritmos e Programação 1, por exemplo, cuja DP não permite
-cursar uma matéria no segundo semestre e quatro matérias no terceiro, o que acaba atrasando um pouco o curso.
+importância de Introdução à Programação, por exemplo, cuja DP não permite
+cursar duas matérias no segundo semestre e consequentemente três matérias no terceiro, o que acaba atrasando um pouco o curso.
+
+Brotip: Tente dar prioridade as matérias da área de matemática (Geometria Analítica e Algébra Linear, Cálculo Diferencial e Integral I e II) pois são matérias fornecidas pelo DFQM (Departamento de Física, Química e Matemática)
+e dificilmente existem turmas extras pelo número limitado de professores, o que acaba dificultando para refaze-las.
 
 \begin{figure}[h]
     \centering
-    \includegraphics[width=\textwidth]{./imagem/requisitos_materias.png}
+    \includegraphics[width=\textwidth]{./imagem/grade.jpeg}
```

Comment:
Você teria o plano de ensino aí? Ainda estou na dúvida se a imagem ficará boa na impressão... :worried: 
Se tiver o PDF talvez seja possíve extrair essa imagem...

Reply:
a qualidade tá zoada ainda, o ideal seria algo vetorizadinho né

Reply:
se for imagem rasterizada tem que ser ao menos 150 dpi para impressão, mas sim, o ideal seria uma imagem vetorizada msm

Reply:
@Cremonildo

Reply:
eu extraí ela do pdf, mas agora eu consegui a original com a Yeda. Ela me direcionou o link e dá pŕa exportar ela com svg, aí eu exportei e subi nesse comentário(tá zipado porque o github não suporta svg)

[Disciplinas Obrigatórias.zip](https://github.com/caccs/Manual-do-Bixo/files/1662498/Disciplinas.Obrigatorias.zip)

Reply:
Não estou conseguindo converter o svg em pdf pra poder botar no manual, só tá faltando isso

Reply:
Esse .svg foi feito por algum outro programa proprietário D:
Não consigo exportar também para .pdf

Reply:
[Disciplinas Obrigatórias.pdf](https://github.com/caccs/Manual-do-Bixo/files/1676212/Disciplinas.Obrigatorias.pdf)
[Disciplinas Obrigatórias.svg.zip](https://github.com/caccs/Manual-do-Bixo/files/1676219/Disciplinas.Obrigatorias.svg.zip)

Adicione também o .svg para futuras modificações.